src/frontend/components/utils/GlossaryTooltip.jsx

```diff
@@ -26,7 +27,7 @@ export default function GlossaryTooltip(props) {
     return null;
   } else {
     return (
-      <Tooltip title={term.definition}>
+      <Tooltip title={parse('<div>' + term.definition + '</div>')}>
```

Comment:
Should we leave a note somewhere in the Add-a-new-glossary modal about how the glossary term needs to be an exact downcase match of the special terms?

Reply:
Good question! I just double checked, and it looks like we set up the logic to match the lowercase word to the lowercase term saved in the database [logic here](https://github.com/throneless-tech/murakami-viz/blob/5c59e1c65e1a1d46875882a369ff2d217423c5ef/src/frontend/components/dashboard/Home.jsx#L154). So it doesn't matter the case of the term, because the JS will search and compare the downcase version of the word. Let me know if you have questions about this!